src/test/java/org/folio/service/mapping/MappingServiceUnitTest.java

```diff
@@ -195,8 +195,8 @@ void shouldMapInstanceHoldingsAndItem_to_marcRecord_whenMappingProfileTransforma
 
   private List<Transformations> createHoldingsAndItemSimpleFieldTransformations() {
     List<Transformations> transformations = new ArrayList<>();
-    transformations.add(createTransformations(CALLNUMBER_FIELD_ID, CALLNUMBER_FIELD_PATH, "900ff$a", HOLDINGS));
     transformations.add(createTransformations(CALLNUMBER_PREFIX_FIELD_ID, CALLNUMBER_PREFIX_FIELD_PATH, "900ff$b", HOLDINGS));
+    transformations.add(createTransformations(CALLNUMBER_FIELD_ID, CALLNUMBER_FIELD_PATH, "900ff$a", HOLDINGS));
```

Comment:
I do not write a separate unit test to verify sorting, just changed a sequence of transformations added. The test passes successfully that confirms the comparator works. I propose to verify all the other cases on the ref env using regular UI application